app/views/items/index.html.erb

```diff
@@ -10,6 +10,10 @@
   <% end %>
 </ul>
 
+<% if current_merchant? %>
+  <%= link_to 'Add New Item', '/dashboard/items/new' %>
```

Comment:
Should prob use path helper here

Reply:
You're totally right - I had forgotten to change the path once I made the route. Changed. Thanks!